dask_expr/expr.py

```diff
@@ -780,6 +781,25 @@ class DropnaFrame(Blockwise):
     operation = M.dropna
 
 
+class RenameFrame(Blockwise):
+    _parameters = ["frame", "columns"]
+    _keyword_only = ["columns"]
+    operation = M.rename
+
+    def _simplify_up(self, parent):
+        if isinstance(parent, Projection) and isinstance(self.columns, Mapping):
```

Comment:
Historically @rjzamora has used `self.operand("columns")` when we wants to refer to an operand that collides with a property name.  I'm surprised that this isn't necessary here.

Reply:
Thx, this is necessary. Test coverage wasn't good enough to show this. Added a test and cleaned this up